### PR TITLE
Add website for MS-AMP

### PR DESCRIPTION
**Pull request by tocean**
**Description**
Add a website for MS-AMP.

**Major Revision**
- Add website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,191 +1,10 @@
...
-the rights to use your contribution. For details, visit [CLA](https://cla.opensource.microsoft.com).
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+## _Check [aka.ms/msamp/doc](https://aka.ms/msamp/doc) for more details._
```
Comment (wkcn) [inline README.md:7]: I think the README can preserve `Features`、`Optimizer Level`、`Performance`、`Contributing` and `Trademarks`.

File: docs/developer-guides/contributing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+1. New features such as support for new distributed training framework.
+
+   If you would like to contribute a new feature on MS-AMP, please submit your proposal first. In [GitHub Issues](https://github.com/azure/MS-AMP/issues) module, choose `Enhancement Request` to finish the submission. If the proposal is accepted, you can submit pull requests to origin `main` branch.
+
+#### Contribution steps
+
+If you would like to contribute to the project, please follow below steps of joint development on GitHub.
+
+1. `Fork` the repo first to your personal GitHub account.
+1. Checkout from main branch for feature development.
```
Comment (wkcn) [inline docs/developer-guides/contributing.md:40]: The order number should be fixed.

File: README.md (hunk 1 above)
Comment (tocean) [reply README.md:7]: All the contents are moved to the website: https://azure.github.io/MS-AMP/docs/introduction

File: docs/developer-guides/contributing.md (hunk 2 above)
Comment (tocean) [reply docs/developer-guides/contributing.md:40]: Using order number in markdown will break some rule in mklint. Always using `1` for all items has same effect in markdown.
https://azure.github.io/MS-AMP/docs/developer-guides/contributing 
image

File: README.md (hunk 1 above)
Comment (wkcn) [reply README.md:7]: I see. Thanks!

**Review by wkcn — APPROVED** [review]
LGTM. Thanks Yuxiang!

Although some information in README has been moved to the website, I think README should show some brief information about the features of MS-AMP to the users.

**Comment by tocean** [comment]
> LGTM. Thanks Yuxiang!
> 
> Although some information in README has been moved to the website, I think README should show some brief information about the features of MS-AMP to the users.

We can discuss this later. Just want to align the README with superbench.

File: docs/developer-guides/contributing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+MS-AMP is an open-source project. Your participation and contribution are highly appreciated. There are several important things you need know before contributing new feature to this project:
+
+#### What content can be added to MS-AMP
+
+1. Bug fixes for existing features.
+1. Performance improvement.
+1. New features such as support for new distributed training framework.
+
+   If you would like to contribute a new feature on MS-AMP, please submit your proposal first. In [GitHub Issues](https://github.com/azure/MS-AMP/issues) module, choose `Enhancement Request` to finish the submission. If the proposal is accepted, you can submit pull requests to origin `main` branch.
```
Comment (guoshzhao) [inline docs/developer-guides/contributing.md:33]: Delete space at the beginning.

**Review by guoshzhao — APPROVED** [review]

File: docs/developer-guides/contributing.md (hunk 3 above)
Comment (tocean) [reply docs/developer-guides/contributing.md]: Fixed.

**Pull request merged**